Library/Homebrew/service.rb

```diff
@@ -141,5 +141,34 @@ def to_plist
 
       base.to_plist
     end
+
+    # Returns a `String` systemd unit.
+    # @return [String]
+    sig { returns(String) }
+    def to_systemd_unit
+      instance_eval(&@service_block)
+
+      clean_command = @run.select { |i| i.is_a?(Pathname) }
+                          .map(&:to_s)
+
+      unit = <<~EOS
+        [Unit]
+        Description=Brew generated Unit for #{@formula.name}
+
+        [Service]
+        Type=simple
+        ExecStart=#{clean_command.join}
+      EOS
+      unit += "\nRestart=always" if @keep_alive == true
```

Comment:
It'd be nice here to build an array of `units << "Restart=always"` and then do `join("\n")` at the end to avoid hardcoding the `\n`